js/CRM/Form/MembersOnlyEventTab.js

```diff
@@ -5,6 +5,10 @@ jQuery(document).ready(function(){
   var LINK_TYPE_CONTRIBUTION_PAGE = '0';
   var LINK_TYPE_URL = '1';
 
+  var EVENT_ACCESS_TYPE_MEMBERS_ONLY = '1';
+  var EVENT_ACCESS_TYPE_GROUPS_ONLY = '2';
+  var EVENT_ACCESS_TYPE_AUTHENTICATED_ONLY = '3';
```

Comment:
See that the form were built with constant values from BAO class.  Shouldn't we assign 1,2,3 from constant to these values, so we can ensure that there will be any regression if the values have changed in the future? 

Similar to below. 

```
 var EVENT_ACCESS_TYPE_MEMBERS_ONLY = {$EVENT_ACCESS_TYPE_MEMBERS_ONLY};
 var EVENT_ACCESS_TYPE_GROUPS_ONLY = '...;
 var EVENT_ACCESS_TYPE_AUTHENTICATED_ONLY = '...';
```

Reply:
Updated to pass the constant values from server-side - see the commit "WHF-308: Pass constant values from server-side".